pkg/server/handler/auth.go

```diff
@@ -53,6 +54,10 @@ func (payload *signupPayload) Validate() skyerr.Error {
 			return skyerr.NewInvalidArgument("invalid auth data", []string{"auth_data"})
 		}
 
+		if dupliatedKeys := payload.duplicatedKeysInAuthDataAndProfile(); len(dupliatedKeys) > 0 {
```

Comment:
Do you think it is better to ignore the keys instead of throwing an error? @rickmak